org.gnome.Dictionary.yml

```diff
@@ -1,6 +1,6 @@
 app-id: org.gnome.Dictionary
 runtime: org.gnome.Platform
-runtime-version: '3.36'
+runtime-version: '40'
 sdk: org.gnome.Sdk
 command: gnome-dictionary
 rename-icon: accessories-dictionary
```

Comment:
You can drop this line, the app ships its own icon now

Reply:
Along with all the icons in this repository

Reply:
Do you mean line 6?

Reply:
Yes

Reply:
Note that as you removed the icons, you also has to remove that part from the manifest
```
  - name: icons
    buildsystem: simple
    build-commands:
      - install -Dm644 accessories-dictionary-24.png /app/share/icons/hicolor/24x24/apps/accessories-dictionary.png
      - install -Dm644 accessories-dictionary-32.png /app/share/icons/hicolor/32x32/apps/accessories-dictionary.png
      - install -Dm644 accessories-dictionary-48.png /app/share/icons/hicolor/48x48/apps/accessories-dictionary.png
      - install -Dm644 accessories-dictionary-64.png /app/share/icons/hicolor/64x64/apps/accessories-dictionary.png
      - install -Dm644 accessories-dictionary-128.png /app/share/icons/hicolor/128x128/apps/accessories-dictionary.png
      - install -Dm644 accessories-dictionary-256.png /app/share/icons/hicolor/256x256/apps/accessories-dictionary.png
      - install -Dm644 accessories-dictionary-512.png /app/share/icons/hicolor/512x512/apps/accessories-dictionary.png
    sources:
      - type: file
        path: accessories-dictionary-24.png
      - type: file
        path: accessories-dictionary-32.png
      - type: file
        path: accessories-dictionary-48.png
      - type: file
        path: accessories-dictionary-64.png
      - type: file
        path: accessories-dictionary-128.png
      - type: file
        path: accessories-dictionary-256.png
      - type: file
        path: accessories-dictionary-512.png
```

Reply:
I am sorry. I am currently a beginner at flatpaks. I fix it.

Reply:
No worries :-)